### PR TITLE
storage: add eth runtime source storage impl 

### PR DESCRIPTION
**Pull request by pro-wh**
if things get too weird with deploying an SGX-enabled node with exposed gRPC, here's this

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -25,6 +25,7 @@ require (
 )
 
 require (
+	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6 // indirect
```
Comment (mitjat) [inline go.mod:28]: Do you know why all the new deps?

**Review by mitjat — COMMENTED** [review]
Oh nice, I didn't realize you got that far along with cleaning this up. We should have started pushing for gRPC access sooner, might have avoided some code churn :|. It was a bad combo of Gregor's and my OOO one after another.

I think no need to merge until we know we need this? But it's nice that it's ready.

File: go.mod (hunk 1 above)
Comment (pro-wh) [reply go.mod:28]: should be from ethclient

**Comment by pro-wh** [comment]
#305 is moved over to a proper sapphire node

**Pull request closed without merging**